R/derive_vars_crit.R

```diff
@@ -0,0 +1,89 @@
+#' derive_vars_crit
```

Comment:
You need a Title for this function

Reply:
https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#function-header-documentation